### PR TITLE
Allow 0 value in cuttlefish schema via "byte" validator

### LINKED ISSUE #2493: "byte" cuttlefish schema validator should allow 0
Reported here:

https://groups.google.com/d/topic/rabbitmq-users/PTnlN9HiCWQ/discussion

### PR DESCRIPTION
**Pull request by lukebakken**
Fixes #2493

### CODE REVIEW

File: priv/schema/rabbit.schema [hunk 1]
```diff
@@ -1756,7 +1756,7 @@ end}.
 
 {validator, "byte", "Integer is not 0<i<255",
```
Comment (michaelklishin) [inline priv/schema/rabbit.schema:1757]: This comment sounds wrong (and in particular now). Should we correct it?

Comment (lukebakken) [reply priv/schema/rabbit.schema:1757]: :facepalm: of course. I'll get it.

Comment (lukebakken) [reply priv/schema/rabbit.schema]: 255 is valid as well. One more sec!

File: priv/schema/rabbit.schema [hunk 2]
```diff
@@ -1754,9 +1754,9 @@ fun(Port) when is_integer(Port) ->
     Port > 0 andalso Port < 65535
 end}.
 
-{validator, "byte", "Integer is not 0<i<255",
+{validator, "byte", "Integer is not 0<=i<=255",
```
Comment (michaelklishin) [inline priv/schema/rabbit.schema:1757]: Should this say "Integer *is* in [0, 255]"?

Comment (lukebakken) [reply priv/schema/rabbit.schema]: Updated!

Comment (michaelklishin) [reply priv/schema/rabbit.schema]: Um, this is the error message, so the negation is expected here. I'll correct.

**Review by michaelklishin — APPROVED** [review]

**Comment by michaelklishin** [comment]
Backported to `v3.8.x`.

**Comment by lukebakken** [comment]
Thank you @michaelklishin

**Pull request merged**